### ISSUE TITLE
Section describing types of GitHub repos and what they are for

### ISSUE THREAD

**Issue by dwillis** [openelections/docs#36]
And how to contribute to them.

**Comment by jotasprout** [comment]
This is a standalone page, correct? If so, where would this go in the site hierarchy? Where would it need to appear in navigation? Content-wise, is this as easy as finding the answers for "what for" and "how to" in each repo type's files? Who could I contact to ask any questions?

**Comment by dwillis** [comment]
Yes, it would be a standalone page. I think I would opt for renaming the current "Results" drop-down nav to "Our Data" and then have this as a sub-page.
